documentation/asciidoc/topics/con_operator_deployments.adoc

```diff
@@ -0,0 +1,22 @@
+//This topic is intended as downstream content.
+[id='infinispan-operator-deployments_{context}']
+= {ispn_operator} deployments
+
+When you install {ispn_operator}, it extends the Kubernetes API with Custom Resources (CR) for deploying and managing {brandname} clusters on {openshift}.
+
+To interact with {ispn_operator}, {openshiftshort} users apply Custom Resource Definitions (CRDs) through the {osweb} or `oc` client.
+{ispn_operator} listens for `Infinispan` CRs and automatically provisions native resources, such as StatefulSets and Secrets, in addition to any resources for your {brandname} deployment.
```

Comment:
Maybe we should clarify what is meant by "any resources" here. I assume you're referring to the downloading of db drivers, user libraries etc

Reply:
I've reworded this to say that the operator also configures Infinispan services according to user CR specifications. That's what I was driving at before. The operator sets up all the k8s native objects in addition to any Infinispan cluster configuration.